src/models/fhir/constraints.ts

```diff
@@ -302,13 +312,14 @@ export const getRecognisedConstraints = (
     "Supervised",
     "Remotely Supervised",
   ];
-  if (type === pdtHealthCertV2.PdtTypes.Art) {
-    for (let i = 0; i < observationCount; i += 1) {
+  for (let i = 0; i < observationCount; i += 1) {
+    if (type === pdtHealthCertV2.PdtTypes.Art) {
```

Comment:
yes, moved it out because now we're also adding iso date time validation for all observations' effectiveDateTime. so the custom ios date time validation is common validation for all types.

Reply:
Ah okay, was hard to tell the indentation over GitHub 🤭